src/templates/fr/2019/accessibility_statement.html

```diff
@@ -0,0 +1,122 @@
+{% extends "base/2019/accessibility_statement.html" %}
+
+{% block title %}Déclaration d’accessibilité | Web Almanac par HTTP Archive{% endblock %}
+
+{% block description %}Décrit les objectifs d’accessibilité du Web Almanac{% endblock %}
+
+{% block twitter_image_alt %}Déclaration d’accessibilité du Web Almanac{% endblock %}
+
+{% block date_published %}2020-02-03T12:00:00.000Z{% endblock %}
+{% block date_modified %}2020-07-29T00:00:00.000Z{% endblock %}
+
+{% block index %}
+      <ul>
+        <li><a href="#overview">Aperçu</a></li>
+        <li><a href="#measures">Measures to support accessibility</a></li>
+        <li><a href="#conformance-status">Statut de conformité</a></li>
+        <li><a href="#external-content">Contenu externe</a></li>
+        <li><a href="#compatibility">Compatibilité avec les navigateurs et les technologies d’assistance</a></li>
+        <li><a href="#technical-specifications">Spécifications techniques</a></li>
+        <li><a href="#assessment-approach">Approche d’évaluation</a></li>
+        <li><a href="#feedback">Retour d’information</a></li>
```

Comment:
Do you think we should change the anchor links? They are auto generated for the chapters so do change when you change the heading for them, but manually set for these non-chapter templates. If changing then will need to also change the ids they link to further down.

Reply:
I hesitated, I didn't know if there was something blocking on it (I was thinking to the lang switcher if there is an anchor, but it seems not). 

If there is no blocker, yes, I'd prefer to have them in french too. Will add them asap

Reply:
No blocker, so go for it!

Reply:
Ok, updated link in https://github.com/HTTPArchive/almanac.httparchive.org/pull/1196/commits/cbd0158f2ce0a737ddd7b1d02b463a257ef1ee23

Reply:
Thanks! Fixed in https://github.com/HTTPArchive/almanac.httparchive.org/pull/1196/commits/d4fb45ccbc6e1f265c7676d347f8fd91f15ea5dc